Update dependencies used in tests

forevermatt opened the pull request:
### Fixed
- Update dependencies used for tests
- Stop trying to configure Logentries
- Stop using `runny`
- Ensure docker compose volume mount destinations exist in vendor folder
- Fix permissions issue related to composer's vendor folder in tests
- Make docker compose volume mounts more closely match codeship services

forevermatt committed:
Stop trying to configure Logentries

We no longer use that service.

forevermatt committed:
Fix permissions issue related to composer's vendor folder in tests

forevermatt committed:
Fix volume mount problem during local testing

forevermatt committed:
Make docker compose volume mounts more closely match codeship services

forevermatt committed:
Exit the tests with an error code if a step fails

forevermatt committed:
Stop using `runny`

forevermatt committed:
Show what commands are being run during tests

forevermatt committed:
Update dependencies used for tests

forevermatt committed:
Ensure docker compose volume mount destinations exist in vendor folder

forevermatt committed:
Resume mounting vendor folder during tests

forevermatt committed:
Raise minimum PHP version to 7.3

briskt commented on Makefile:
```diff
@@ -17,6 +17,12 @@ clean:
 
 composer:
 	docker-compose run --rm composer bash -c "composer install --no-scripts"
+	touch vendor/simplesamlphp/simplesamlphp/config/authsources.php
```
Do you still need these `touch` commands since you reverted the `vendor` volume change?

forevermatt replied:
Yeah, those `touch` commands are what enabled me to revert the `vendor` volume change. I think it has to do with the way we're volume-mounting individual files into the vendor/simplesamlphp folder AND we're volume-mounting the entire vendor folder... those files end up being created as directories if they don't already exist, so I need to `touch` them after installing the composer dependencies.

forevermatt commented:
The failed build is because our codeship tests use ssp-base:latest (still running PHP 7.2), but these dependencies are locally tested using ssp-base:develop (as part of updating everything to PHP 7.4). I may need to wait to merge this one until after releasing ssp-base with PHP 7.4.

forevermatt committed:
Remove "version" line in codeship-services.yml, since it's ignored

See:
https://docs.cloudbees.com/docs/cloudbees-codeship/latest/pro-builds-and-configuration/services

forevermatt committed:
Allow SimpleSAMLphp 1.19, stop allowing < 1.18.6, update dependencies

forevermatt commented:
This is so old I'm simply going to close it.
